config/schemas/config.yaml

```diff
@@ -1473,6 +1473,18 @@ properties:
             title: Enable Kafka
             type: boolean
         additionalProperties: false
+      extraRoles:
+        title: Configure extra Roles for application developers
+        type: object
+      extraRoleBindings:
+        title: Configure extra RoleBindings for application developers
+        type: object
+      extraClusterRoles:
+        title: Configure extra ClusterRoles for application developers
+        type: object
+      extraClusterRoleBindings:
+        title: Configure extra ClusterRoleBindings for application developers
+        type: object
```

Comment:
Describe how these are supposed to be use, because right now one would have to look at the values templates for user-crds to see how it is setup.
And that the resulting roles and rolebindings are in all application developer namespaces.